specification/ws/src/main/scripts/org/kaazing/specification/ws/opening/response.header.sec.websocket.extensions.partial.agreement/handshake.request.rpt

```diff
@@ -0,0 +1,20 @@
+connect http://localhost:8080/path?query
```

Comment:
Oh I see, it's just that you're using the Sec-WebSocket-Protocol header. But should be Sec-WebSocket-Extensions  according to the RFC(see my next comment).